lib/quake_parser.ex

```diff
@@ -3,42 +3,120 @@ defmodule QuakeParser do
   `QuakeParser` is a parser for Quake game logs.
   """
 
-  alias QuakeParser.{Game, FindPlayers, KillsOfPlayer}
+  defstruct total_kills: 0, players: [], kills: %{}
 
+  @doc """
+  Parse the Quake log file, and return a QuakeParser object for each game.
```

Comment:
Coloca um exemplo na doc de qual assim:


```suggestion
  Parse the Quake log file, and return a QuakeParser object for each game.
  
  ## Examples

      iex> start(...)
      [...]
```